src/core/settings.h

```diff
@@ -90,6 +90,8 @@ typedef struct QUIC_SETTINGS_INTERNAL {
     uint16_t MaxBindingStatelessOperations;
     uint16_t StatelessOperationExpirationMs;
     uint16_t CongestionControlAlgorithm;
+    const uint32_t* RawDataPathProcList;
+    uint32_t RawDataPathProcListLength;
```

Comment:
Store this directly on MsQuicLib, not the settings (which are duplicated per connection).